### PR TITLE
Add Perceiver IO demo

### PR DESCRIPTION
**Pull request by milank94**
Test by running the following command from `model_demos` on latest alpha release (v0.10.9-alpha):

```bash
python3 -m pytest tests/test_pytorch_perceiverio.py
```

### CODE REVIEW

File: model_demos/README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ python cv_demos/resnet/pytorch_resnet.py
 |   [OpenPose](cv_demos/openpose/)          |     GS, WH   |
 |   [OPT](nlp_demos/opt/)                  |     GS, WH   |
 |   [Pose Landmark](cv_demos/landmark/)  |    WH   |
+|   [Perceiver IO](cv_demos/perceiverio/) |     GS, WH   |
```
[the remark this answers is not in the document]
Comment (milank94) [reply model_demos/README.md]: Good idea. Check the changes from the latest commit: [f19c203b0caa717ee46bc4c39763b28185c4a9a2](https://github.com/tenstorrent/tt-buda-demos/pull/40/files#diff-c8f177750439bdbcda07a63e6ef9dc27b5787cf99466d3d0448da1bf86c92972L30)

I would like to add links to the releases as well. The challenge is that the GS and WH are different links. I might just add this as a hyperlink under the Supported Hardware column

**Comment by anirudTT** [comment]
Test the model test_pytorch_perceiverio.py on the alpha release (v0.10.9-alpha):

E75 Grayskull
- [ ] TT E75- Ubuntu 20.04 
- [ ] TT E75- Ubuntu 22.04 
- [ ] TT E75 - Docker image Ubuntu 20.04
- [ ] TT E75- Docker image Ubuntu 22.04

E150 Grayskull
- [ ] TT E150- Ubuntu 20.04 
- [ ] TT E150- Ubuntu 22.04 
- [ ] TT E150- Docker image Ubuntu 20.04
- [ ] TT E150- Docker image Ubuntu 22.04

N150 Wormhole
- [ ] TT N150- Ubuntu 20.04 
- [ ] TT N150- Ubuntu 22.04 
- [ ] TT N150- Docker image Ubuntu 20.04
- [ ] TT N150- Docker image Ubuntu 22.04

**Review by anirudTT — APPROVED** [review]
LGTM

**Review by nvukobratTT — APPROVED** [review]
Looks good! :)))

**Pull request merged**